draft-mouris-cfrg-mastic.md

```diff
@@ -237,9 +285,98 @@ This document uses the following terms as defined in {{!VDAF}}:
 "prep share", and
 "report".
 
-Mastic uses finite fields as specified in {{Section 6.1 of !VDAF}}. We usually
-denote a finite field by `F` and its Python class object, a subclass of
-`Field`, as `field: type[F]`.
+Mastic uses finite fields of prime order as specified in {{Section 6.1 of
+  !VDAF}}. We usually denote a finite field by `F` and its Python class object,
+  a subclass of `Field`, as `field: type[F]`. The following basic API functions
+  referenced in this document are as defined therein:
+
+* `Field(integer: int) -> Field` returns an integer represented as a field
+  element. The value of integer MUST be non-negative and less than
+  Field.MODULUS.
+* `Field.as_unsigned() -> int` returns the integer representation of a field
```

Comment:
* This has been changed to `int()` in the upcoming draft.
* We use capitalization to distinguish between different types of Python object: `Field` indicates this is a class; `field` would indicate an instance of that class. (Though `elem` may be better variable name)